synapse/cortex.py

```diff
@@ -1218,6 +1227,104 @@ async def _initCoreQueues(self):
 
         self.multiqueue = await slab.getMultiQueue('cortex:queue', nexsroot=self.nexsroot)
 
+    async def _initSubscribers(self):
```

Comment:
Kinda digging editWatchers or editMonitors, but this is just a placeholder comment to make sure we decide and make the name use consistent.